src/status_im2/common/scroll_page/style.cljs

```diff
@@ -58,7 +58,7 @@
   [animation]
   (reanimated/apply-animations-to-style
    {:transform [{:scale animation}]}
-   {:border-radius    40
+   {:border-radius    50
```

Comment:
It's nice that you found a fix, but I must say I don't understand why the shape was oval if `40` is half of the width and height of the display picture `80`. Something should be going on, because we have lots of places in the code where the formula `border-radius = square-side/2` gives a perfect circle.

Reply:
Here, the element's width is not just 80, it's 80 + 4 (with 2 pixels of padding on each side). As a result, a border radius of 40 won't result in a perfect circle. A border radius of 44 would have been sufficient, but to account for slight variations in rendering on different platforms, such as iOS, a border radius of 50 was used for safety.

Reply:
Oh that's true, thanks for the explanation @ajayesivan.